engine/runtime/src/test/java/org/enso/example/TestClass.java

```diff
@@ -19,6 +20,32 @@ public static long add(long a, long b) {
     return a + b;
   }
 
+  public static double doubleArrayAverage(double[] arr) {
```

Comment:
The `averageOfMixedArrayOverDouble` test prepares array of power of 2 with 200 elements. Some of them are `Long`, some of them are `EnsoBigInteger`.

However the Java side can request them to be converted to `double` - useful for example for statistic functions where the precision doesn't matter much.